### PR TITLE
Event model

### PR DESCRIPTION
**Pull request by klove2016**

### CODE REVIEW

File: app/views/events/_event.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1,2 @@
+json.extract! event, :id, :title, :description, :user_id, :created_at, :updated_at
+json.url event_url(event, format: :json)
```
Comment (baileygriff) [inline app/views/events/_event.json.jbuilder:2]: What is this json code for?

File: app/views/events/index.json.jbuilder [hunk 2]
```diff
@@ -0,0 +1 @@
+json.array! @events, partial: "events/event", as: :event
```
Comment (baileygriff) [inline app/views/events/index.json.jbuilder:1]: Same here, what's this for?

File: app/views/events/show.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1 @@
+json.partial! "events/event", event: @event
```
Comment (baileygriff) [inline app/views/events/show.json.jbuilder:1]: And here?

File: spec/views/events/edit.html.erb_spec.rb
Comment (baileygriff) [inline spec/views/events/edit.html.erb_spec.rb:1]: I've never even seen ERB specs before. I think we can leave them but probably not really necessary to write more. We can discuss testing some more in the next meeting. There's a lot of personal opinion about what pieces of code should or shouldn't be tested

**Review by baileygriff — APPROVED** [review]
Other than the changes we made during the call, this all looks good to me

File: app/views/events/_event.json.jbuilder (hunk 1 above)
Comment (klove2016) [reply app/views/events/_event.json.jbuilder:2]: Al this json code came when I ran the rails generate scaffold Events

**Pull request merged**